pallets/offchain-worker/src/tests.rs

```diff
@@ -157,17 +166,17 @@ impl ExternalityBuilder {
 #[test]
 fn test_chars_to_u128() {
 	let correct_balance = vec!['5', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0'];
-	assert_eq!(Ok(500000000000000000_u128), <Module<TestRuntime>>::chars_to_u128(&correct_balance));
+	assert_eq!(Ok(500000000000000000_u128), utils::chars_to_u128(&correct_balance));
```

Comment:
When you move functions (refactor), it's better to mention it in the pr comments